src/routes/Private.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { BrowserRouter, Link, Switch, Route } from 'react-router-dom';
```

Comment:
It looks like my changes got dropped after we merged my branch into yours yesterday.

Reply:
I'll put up a pr with that as the diff